tests/checkpoints_test.py

```diff
@@ -258,10 +258,11 @@ def test_convert_checkpoint(self):
         flax.core.unfreeze(
             checkpoints.convert_pre_linen(state_dict['optimizer'])))
     optimizer = optimizer.apply_gradient(variables['params'])
-    batch_stats= flax.core.freeze(flax.traverse_util.unflatten_dict({
-        tuple(k.split('/')[1:]): v
-        for k, v in checkpoints.convert_pre_linen(model_state.as_dict()).items()
-    }))
+    batch_stats = checkpoints.convert_pre_linen(
```

Comment:
While writing the doc I noticed that the order of conversion was wrong!

Is it expected that Linen doesn't complain when the mutable collections (`batch_stats` in this case) don't match?

Reply:
ping @jheek